app/src/main/java/com/example/taskyy/ui/events/RegisterEvent.kt

```diff
@@ -1,12 +1,14 @@
 package com.example.taskyy.ui.events
 
+import android.content.Context
+
 
 sealed interface RegisterEvent{
     data class OnNameChanged(val name: String): RegisterEvent
     data class OnEmailChanged(val email: String): RegisterEvent
     data class OnPasswordChanged(val password: String): RegisterEvent
     data class OnTogglePasswordVisibility(val isPasswordVisible: Boolean): RegisterEvent
 
-    data object OnGetStartedClick: RegisterEvent
+    data class OnGetStartedClick(val applicationContext: Context) : RegisterEvent
```

Comment:
I wouldn't recommend to pass the context to a ViewModel. On the one hand, it can quickly happen that you run into memory leaks in the VM when saving it in a field there (not with the app context, but with the activity context).
But even with the application context, you have the issue that your ViewModel suddenly relies on the Android SDK. In case you'd want to write a test case for it later, that would only work when it's an instrumented test (which is a test that has to run on an actual Android device and is much slower) rather than a local unit test (which is a test that simply runs on the JVM directly and doesn't depend on Android)